ribbon/poetry.lock

```diff
@@ -0,0 +1,1291 @@
+[[package]]
```

Comment:
it seems we need to ignore `poetry.lock` in all folders because they are not needed for now